### PR TITLE
Maintain trait functions ordering.

### PR DESCRIPTION
**Pull request by kchalkias**
##  Motivation

It's a good practice, for better readability, to maintain the order of defined trait functions during implementation.

### Have you read the [Contributing Guidelines on pull requests](https://github.com/libra/libra/blob/master/CONTRIBUTING.md#pull-requests)?

YES

## Test Plan

-

### CODE REVIEW

**Comment by kchalkias** [comment]
@valerini just rebased and resolved conflicts, back to you.

File: storage/libradb/src/event_store/mod.rs [hunk 1]
```diff
@@ -114,7 +114,6 @@ impl EventStore {
                 // Queries tend to base on very recent ledger infos, so first try to linear search
                 // from the most recent end, for limited tries.
                 // TODO: Optimize: Physical store use reverse order.
-                let mut n_try_recent = 10;
```
Comment (zekun000) [inline storage/libradb/src/event_store/mod.rs]: Why did you remove this line? seems not related to this PR.

This breaks `cargo test -p consensus`

Comment (kchalkias) [reply storage/libradb/src/event_store/mod.rs]: Hm, didn't notice the implication, it's from automation tool fix, will submit a revert PR for this line. Thanks!

Comment (zekun000) [reply storage/libradb/src/event_store/mod.rs]: Thanks for confirming, It's fixed in https://github.com/libra/libra/pull/543
